tallycash/extension
# 📡 Fix populating past transaction activity list

Resolves https://github.com/tallycash/extension/issues/1439

### What
Past transactions are fetched in batches - we should fetch only 5 transactions at once even if we know more transaction hashes.

### Fix
Filtering transactions hashes list has been fixed because `filter` was not waiting for `async`, promisified arrow function.

### To test
1. Check given account's activity on Etherscan (to know how many transactions expect)
2. Add this account to the wallet (preferred fresh install)
3. Go to the activity list and wait 5-10min - depending on how old transactions are
4. Activity list should be slowly populated with transactions.

![image](https://user-images.githubusercontent.com/20949277/171418408-c5af73f7-7656-4b23-ac77-6ab99e46396b.png)
